src/components/game/Sketch.tsx

```diff
@@ -198,6 +85,55 @@ const SketchComponent = ({ game, evaluation }: SketchProps) => {
     boardGroundPath && sketch.image(imageFromPath(boardGroundPath), 0, 0, sketch.width, sketch.height)
   }
 
+  const loadedSounds: Map<Id, { lastStatus: string, soundFile: p5.SoundFile, started: boolean }> = new Map()
+
+  function playSounds() {
+    currentSoundStates(evaluation).forEach((soundState: SoundState) => {
+      if (!loadedSounds.has(soundState.id)) {
+        loadedSounds.set(soundState.id,
+          {
+            lastStatus: soundState.status,
+            soundFile: new p5.SoundFile(game.assetsDir + soundState.file),
+            started: false,
+          })
+      }
+
+      const loadedSound = loadedSounds.get(soundState.id)!
+      loadedSound.soundFile.setLoop(soundState.loop)
+      loadedSound.soundFile.setVolume(soundState.volume)
+
+      if ((loadedSound.lastStatus !== soundState.status && loadedSound.soundFile.isLoaded()) || (!loadedSound.started && loadedSound.soundFile.isLoaded())) {
```

Comment:
```suggestion
      if ( (loadedSound.lastStatus !== soundState.status || !loadedSound.started) && loadedSound.soundFile.isLoaded() ) {
```

Estaría bueno sacar esta condición a otra función `soundChanged`.